velox/functions/prestosql/aggregates/BitwiseAggregates.cpp

```diff
@@ -98,48 +98,6 @@ class BitwiseAndAggregate : public BitwiseAggregateBase<T> {
   }
 };
 
-template <template <typename U> class T>
-bool registerBitwise(const std::string& name) {
-  // TODO Fix the signatures to match Presto.
-  std::vector<std::shared_ptr<exec::AggregateFunctionSignature>> signatures;
-  for (const auto& inputType : {"tinyint", "smallint", "integer", "bigint"}) {
-    signatures.push_back(exec::AggregateFunctionSignatureBuilder()
-                             .returnType(inputType)
-                             .intermediateType(inputType)
-                             .argumentType(inputType)
-                             .build());
-  }
-
-  exec::registerAggregateFunction(
-      name,
-      std::move(signatures),
-      [name](
-          core::AggregationNode::Step step,
-          const std::vector<TypePtr>& argTypes,
-          const TypePtr& resultType) -> std::unique_ptr<exec::Aggregate> {
-        VELOX_CHECK_LE(argTypes.size(), 1, "{} takes only one argument", name);
-        auto inputType = argTypes[0];
-        switch (inputType->kind()) {
-          case TypeKind::TINYINT:
-            return std::make_unique<T<int8_t>>(resultType);
-          case TypeKind::SMALLINT:
-            return std::make_unique<T<int16_t>>(resultType);
-          case TypeKind::INTEGER:
-            return std::make_unique<T<int32_t>>(resultType);
-          case TypeKind::BIGINT:
-            return std::make_unique<T<int64_t>>(resultType);
-          default:
-            VELOX_CHECK(
-                false,
-                "Unknown input type for {} aggregation {}",
-                name,
-                inputType->kindName());
-        }
-      },
-      true);
```

Comment:
Hi @Yohahaha, do we need this line？
I found that https://github.com/facebookincubator/velox/blob/main/velox/functions/lib/aggregates/BitwiseAggregateBase.h#L83-L109 do not have this line.